spec/features/landing_pages_page_spec.rb

```diff
@@ -72,6 +72,9 @@ def expect_elements(page, landing_page)
   end
 
   def expect_page_text(string)
+    return unless string
+    raise "Found invalid page text: '#{string.inspect}' of class '#{string.class}'" unless string.is_a?(String)
```

Comment:
Yep, that's what I meant.

Reply:
Yeah, the benefit to this is if this is a `Hash…` here, it means localization failed, so it would render as `{ 'en-NZ': … }` in the HTML.

Might move that check a bit further up—eg. on the landing page model—but these problems are really just because we're like 2-3 years behind on versions.